packages/css/src/atoms/avatar/_index.scss

```diff
@@ -0,0 +1,18 @@
+@use "../../themes/%avatar%/avatar" as tokens;
+@use "../../utility/index" as utils;
+
+.eds-avatar {
+  @include utils.centered-flex();
+  width: tokens.$width;
+  height: tokens.$height;
+  border-radius: tokens.$border-radius;
+  background-color: tokens.$fill;
+  color: tokens.$font-color;
+  font-family    : tokens.$font-name;
+  font-weight    : tokens.$font-weight;
+  font-size      : tokens.$font-size;
+  line-height    : tokens.$line-height;
+  letter-spacing : tokens.$spacing;
+  user-select: none;
+  cursor: pointer;
```

Comment:
[optional] you can order the properties alphabetically and try to keep a code style between:

```
$button-radius              : global_shape.$pill-radius;
$checkbox-radius         : global_shape.$round-radius;
$button-radius              : global_shape.$pill-radius;
$checkbox-radius         : global_shape.$round-radius;
```

and 

```
$button-radius: global_shape.$pill-radius;
$checkbox-radius: global_shape.$round-radius;
$button-radius: global_shape.$pill-radius;
$checkbox-radius: global_shape.$round-radius;
```

but not mix both.

Reply:
Forgot to run linter. Nice catch!

Reply:
As for the order, there is an order there. First the token attributes followed by non-token attributes. The token attributes are in the same order as the the theme file (which is ordered by Size -> Shape -> Color -> Font)